src/templates/common/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  transformBreadcrumbs,
+  deriveLastBreadcrumbFromPath,
+  deriveLcBreadcrumbs,
+  filterInvalidCrumbs,
+} from '@/templates/globals/util/breadcrumbUtils'
+import { BreadcrumbItem } from '@/types/index'
+
+interface BreadcrumbProps {
+  breadcrumbs?: BreadcrumbItem[]
+  breadcrumbsOverride?: BreadcrumbItem[]
+  breadcrumbTitle?: string
+  disableAnalytics?: boolean
+  deriveBreadcrumbsFromUrl?: boolean
+  replaceLastItem?: boolean
+  constructLcBreadcrumbs?: boolean
+  titleInclude?: boolean
+  hideHomeBreadcrumb?: boolean
+  customCrumbHomeText?: string
+  entityPath?: string
+}
+const Breadcrumbs = ({
+  breadcrumbs,
+  breadcrumbsOverride,
+  entityPath,
+  breadcrumbTitle,
+  disableAnalytics,
+  deriveBreadcrumbsFromUrl,
+  replaceLastItem,
+  constructLcBreadcrumbs,
+  titleInclude,
+  hideHomeBreadcrumb,
+  customCrumbHomeText,
+}: BreadcrumbProps) => {
+  const breadcrumbData: BreadcrumbItem[] = breadcrumbs
+  let crumbs = breadcrumbsOverride || breadcrumbData
```

Comment:
**Question**
Why do we need `breadcrumbData`? Couldn't we have `let crumbs = breadcrumbsOverride || breadcrumbs` ?

Reply:
**Question**
More fundamentally, what's the purpose of the override prop in the first place? It seems to me like any desire to override could be accomplished by simply passing the override data in the `breadcrumbs` prop. It doesn't seem like there is any sort of cascading going on between the override and the base prop. Couldn't the caller of this component just pass the override data as _the_ breadcrumb data?

Reply:
yeah thats a good catch, I was parroting content-build there and didnt consider that we dont need that prop at all.  Will adjust